power_grid_model_c/power_grid_model/include/power_grid_model/common/exception.hpp

```diff
@@ -137,13 +137,6 @@ class InvalidCalculationMethod : public CalculationError {
     InvalidCalculationMethod() : CalculationError("The calculation method is invalid for this calculation!") {}
 };
 
-class UnknownAttributeName : public PowerGridError {
```

Comment:
this error was never thrown (most likely outdated since switching to new metadata structure)

Reply:
Would this line be further processed in this PR?

Reply:
Nope, removed entirely. My comment was just an FYI